lib/ts/framework/express/framework.ts

```diff
@@ -129,6 +134,24 @@ export class ExpressResponse extends BaseResponse {
         setCookieForServerResponse(this.response, key, value, domain, secure, httpOnly, expires, path, sameSite);
     };
 
+    clearCookie = (key: string) => {
```

Comment:
im not sure if this is the way to clear cookies.. clearing cookies requires setting the cookie to an expired date. If instead you meant removeSetCookieFromResponse, then you should change the name of this funciton perhaps.